CMakeLists.txt

```diff
@@ -158,8 +164,80 @@ if(OCOS_ENABLE_SELECTED_OPLIST)
   include(_selectedoplist)
 endif()
 
+set(_OCOS_EXCEPTIONS_REQUIRED OFF)
```

Comment:
Given we only disable exception flags for the non-exception operation, do we still need _OCOS_EXCEPTIONS_REQUIRED since this flag requires the maintenance of the build option list.